Gopkg.toml

```diff
@@ -0,0 +1,48 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+[[constraint]]
+  name = "github.com/fatih/color"
+  version = "^1.5.0"
+
+[[constraint]]
+  name = "github.com/go-sql-driver/mysql"
+  version = "^1.3.0"
+
+[[constraint]]
+  name = "github.com/hashicorp/consul"
+  version = "^0.9.2"
+
+[[constraint]]
+  name = "github.com/percona/pmgo"
+  version = "^0.5.1"
+
+[[constraint]]
+  name = "github.com/prometheus/client_golang"
+  version = "^0.8.0"
+
+[[constraint]]
+  name = "github.com/stretchr/testify"
+  version = "^1.1.4"
+
+[[constraint]]
+  name = "gopkg.in/DATA-DOG/go-sqlmock.v1"
+  version = "^1.2.0"
```

Comment:
Just to be sure, was it really necessary to lock so many versions? Or it was automatic conversion from glide.lock?

Reply:
Yeah, all those packages were locked in Glide too. We will check and remove unnecessary locks early in PMM 1.8 development cycle.